<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YTDownloader</class>
 <widget class="QMainWindow" name="YTDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>INTTube</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Youtube-512.png</normaloff>Youtube-512.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LinkHere">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Link:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LinkedYoutube"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>110</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="updateYoutubeDl">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Aktualizacja</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="ApplyChange">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pobierz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>233</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="VideoOnly">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wideo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Video">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="viewQualityVideo">
         <item>
          <property name="text">
           <string>Najlepsza jakość (MP4)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Najlepsza jakość (MKV)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Najlepsza jakość (WEBM)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Średnia jakość (MP4)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Średnia jakość (MKV)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Średnia jakość (WEBM)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Najgorsza jakość (MP4)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Najgorsza jakość (MKV)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Najgorsza jakość (WEBM)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="MP3Only">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MP3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="MP3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="viewQualityMP3">
         <item>
          <property name="text">
           <string>Najlepsza jakość</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Średnia jakość</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Najgorsza jakość</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>124</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="ClickableLogo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ytd_resource.qrc">:/new/YT-Downloader/logo.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ytd_resource.qrc"/>
 </resources>
 <connections/>
</ui>
